### PR TITLE
Update billing cycle

### PR DESCRIPTION
**Pull request by stephaurelio**
### Proposed changes

This PR updates the billing cycle docs.

Deploy preview: https://deploy-preview-18836--docsdocker.netlify.app/billing/cycle/

### Related issues (optional)

Closes JIRA https://docker.atlassian.net/browse/ENGDOCS-1884

### CODE REVIEW

File: content/billing/cycle.md [hunk 1]
```diff
@@ -6,10 +6,27 @@ keywords: billing, cycle, payments, subscription
 
 You can pay for a Pro or Team subscription on a monthly or yearly billing cycle. You select your preferred billing cycle when you buy your subscription.
 
-You can't change the billing cycle from monthly to annual, or from annually to monthly.
+If you have a monthly billing cycle, you can choose to switch to an annual billing cycle. Note that it's not possible to switch from an annual billing cycle to monthly.
```
Comment (weswhite) [inline content/billing/cycle.md:9]: should we note here that support could do this for them? or do we not want to publish that information?

**Review by weswhite — APPROVED** [review]
looks good

File: content/billing/cycle.md (hunk 1 above)
Comment (stephaurelio) [reply content/billing/cycle.md:9]: I can check with them to see if the request comes up a lot and add this in later if so!

File: content/billing/cycle.md [hunk 2]
```diff
@@ -6,10 +6,27 @@ keywords: billing, cycle, payments, subscription
...
 
-For questions about your billing cycle, [contact support](https://hub.docker.com/support/contact/).
+When you change the billing cycle's duration:
+- The next billing date reflects the new cycle. To check the billing date, see [FAQs](faqs.md#where-can-i-view-my-billing-date).
+- The subscription's start date resets. For example, if the start date of the monthly subscription is March 1st and the end date is April 1st, then after switching the billing duration to March 15th 2023 the new start date is March 15th, 2023, and the new end date is March 15th, 2024.
+- Any unused monthly subscription is prorated and applied as credit towards the new annual period. For example, if the monthly subscription cost is $10 per month and the annual cost is $100 per year, then the unused value of the monthly subscription is $5. The migration cost would be $95 ($100 - $5). The renewal cost after March 15, 2024 is $100.
```
Comment (aevesdocker) [inline content/billing/cycle.md:14]:
```suggestion
- Any unused monthly subscription is prorated and applied as credit towards the new annual period. For example, if you switch from a $10 monthly subscription to a $100 annual plan, deducting the unused monthly value (in this case $5), the migration cost becomes $95 ($100 - $5). The renewal cost after March 15, 2024 is $100.
```
Nit: non-blocking slightly clearer phrasing

**Review by aevesdocker — DISMISSED** [review]
Nice

File: content/billing/cycle.md (hunk 2 above)
Comment (stephaurelio) [reply content/billing/cycle.md]: Left my comment agreeing with you as the commit message lol. Updating!

**Pull request merged**